lib/util/read_repo_ast_util.js

```diff
@@ -36,6 +36,7 @@
  */
 
 const assert   = require("chai").assert;
+const async     = require("generator-async");
```

Comment:
I think `async.run(foo)` is the equivalent of `co(foo)`. If `async` provides something that `co` doesn't, feel free to keep using `async`.